client/ionic/src/components/TopNav.tsx

```diff
@@ -3,21 +3,45 @@ import { IonHeader, IonItem, IonImg, IonToolbar } from '@ionic/react';
 import { IonButtons, IonButton, IonIcon } from '@ionic/react';
 import { close } from 'ionicons/icons';
 
-const TopNav: React.FC = () => {
+const TopNav = ({
+  showClose,
+  linkify,
+}: {
+  showClose?: boolean;
+  linkify?: boolean;
+}) => {
+  const linkProps = linkify ? { routerLink: '/menu' } : {};
   return (
     <IonHeader className="ion-no-border">
       <IonToolbar>
-        <IonItem lines="none" routerLink="/menu">
-          <IonImg className="left h3" src="assets/identity/who-logo-rgb.png" />
-        </IonItem>
-        <IonButtons slot="end">
-          <IonButton routerLink="/menu">
-            <IonIcon slot="end" icon={close} />
+        <IonButtons slot="start">
+          <IonImg
+            className="h3 ml4"
+            style={{ objectPosition: 'left' }}
+            src="assets/identity/who-logo-rgb.png"
+          />
+        </IonButtons>
+        <IonButtons
+          slot="end"
+          style={showClose ? {} : { visibility: 'hidden' }}
+        >
+          <IonButton {...linkProps}>
+            <IonIcon
+              slot="end"
+              icon={close}
+              color="primary"
+              style={{ width: '2em', height: '2em' }}
```

Comment:
a TODO for a CSS style guide.  when do we use inline literal styles, inline shared style objects, or classes for example

Reply:
actually file a issue no todo needed